applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+        </group>
+        <group name="coordinate_system_set" type="NXcoordinate_system_set" minOccurs="1" maxOccurs="1">
+            <doc>
+                Set to hold different coordinate systems conventions.
+                
+                Inspect the description of the :ref:`NXcoordinate_system_set` and :ref:`NXcoordinate_system` base classes
+                how to define coordinate systems in NeXus.
+            </doc>
+            <group type="NXcoordinate_system" minOccurs="1" maxOccurs="unbounded">
+                <field name="alias" type="NX_CHAR" optional="true"/>
+                <field name="type" type="NX_CHAR"/>
+                <field name="handedness" type="NX_CHAR"/>
+                <field name="origin" type="NX_CHAR"/>
+            </group>
+            <group name="processing_reference_frame" type="NXcoordinate_system" recommended="true">
+                <field name="alias" type="NX_CHAR" optional="true"/>
+                <field name="type" type="NX_CHAR"/>
+                <field name="handedness" type="NX_CHAR"/>
+                <field name="origin" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Location of the origin of the processing_reference_frame.
+                        
+                        It is assumed that regions-of-interest in this reference frame form a rectangle or cuboid.
+                        Edges are interpreted by inspecting the direction of their outer unit normals
+                        (which point either parallel or antiparallel) along respective base vector direction
+                        of the reference frame.
+                        
+                        If any of these assumptions is not met, the user is required to explicitly state this.
+                    </doc>
+                    <enumeration>
+                        <item value="front_top_left"/>
+                        <item value="front_top_right"/>
+                        <item value="front_bottom_right"/>
+                        <item value="front_bottom_left"/>
+                        <item value="back_top_left"/>
+                        <item value="back_top_right"/>
+                        <item value="back_bottom_right"/>
+                        <item value="back_bottom_left"/>
+                    </enumeration>
+                </field>
+                <field name="x_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing x-axis base vector of the
+                        processing_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
```

Comment:
Cardinal words?  I recognize this may be part of #1415 which I haven't reviewed yet.

Reply:
Yes, but relaxed into an open enumeration